features/0023-did-exchange/README.md

```diff
@@ -143,19 +143,23 @@ The _requester_ may provision a new DID according to the DID method spec. For a
   "label": "Bob",
   "did": "B.did@B:A",
   "did_doc~attach": {
-     "base64": "eyJ0eXAiOiJKV1Qi... (bytes omitted)",
-     "jws": {
-        "header": {
-           "kid": "did:key:z6MkmjY8GnV5i9YTDtPETC2uUAW6ejw3nk5mXF5yci5ab7th"
-        },
-        "protected": "eyJhbGciOiJFZERTQSIsImlhdCI6MTU4Mzg4... (bytes omitted)",
-        "signature": "3dZWsuru7QAVFUCtTd0s7uc1peYEijx4eyt5... (bytes omitted)"
+      "@id": "B.did@B:A",
```

Comment:
Although this example is non-normative, I don't think it's a good idea to add `@id` and set its value to the DID's ID. That will probably mislead developers.

Reply:
I was certain on what to put there.  What do you suggest?

Reply:
Any UUID would suffice.

Reply:
Done